From 6d4860b1de42a4787f7da39a8e3bc61c120045ad Mon Sep 17 00:00:00 2001
Message-ID: <6d4860b1de42a4787f7da39a8e3bc61c120045ad.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 01:23:16 +0100
Subject: [PATCH v3 05/66] gnu: go-github-com-andybalholm-cascadia: Move to
 (gnu packages golang-web).

* gnu/packages/golang.scm (go-github-com-andybalholm-cascadia): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I185cb923d21685eb5a3952b9990833d3a0709312
---
 gnu/packages/golang-web.scm | 25 +++++++++++++++++++++++++
 gnu/packages/golang.scm     | 24 ------------------------
 2 files changed, 25 insertions(+), 24 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 39a83990ce..3c1abfff2c 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -2,6 +2,7 @@
 ;;; Copyright © 2018 Pierre-Antoine Rouby <pierre-antoine.rouby@inria.fr>
 ;;; Copyright © 2020 Martin Becze <mjbecze@riseup.net>
 ;;; Copyright © 2020 Oleg Pykhalov <go.wigust@gmail.com>
+;;; Copyright © 2022 Giacomo Leidi <goodoldpaul@autistici.org>
 ;;;
 ;;; This file is part of GNU Guix.
 ;;;
@@ -35,6 +36,30 @@ (define-module (gnu packages golang-web)
 ;;;
 ;;; Code:
 
+(define-public go-github-com-andybalholm-cascadia
+  (package
+    (name "go-github-com-andybalholm-cascadia")
+    (version "1.3.1")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/andybalholm/cascadia")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0zgc9fjkn7d66cnmgnmalr9lrq4ii1spap95pf2x1hln4pflib5s"))))
+    (build-system go-build-system)
+    (arguments
+     `(#:import-path "github.com/andybalholm/cascadia"))
+    (native-inputs
+     (list go-golang-org-x-net))
+    (home-page "https://github.com/andybalholm/cascadia/")
+    (synopsis "CSS selectors for HTML")
+    (description "The Cascadia package implements CSS selectors for use with
+the parse trees produced by the html package.")
+    (license license:bsd-2)))
+
 (define-public go-github-com-aymerick-douceur
   (package
     (name "go-github-com-aymerick-douceur")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 3aad98c9ef..a6365a9686 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -9330,30 +9330,6 @@ (define-public go-github-com-alecthomas-chroma
 converts it into syntax highlighted HTML, ANSI-coloured text, etc.")
     (license license:expat)))
 
-(define-public go-github-com-andybalholm-cascadia
-  (package
-    (name "go-github-com-andybalholm-cascadia")
-    (version "1.3.1")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/andybalholm/cascadia")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "0zgc9fjkn7d66cnmgnmalr9lrq4ii1spap95pf2x1hln4pflib5s"))))
-    (build-system go-build-system)
-    (arguments
-     `(#:import-path "github.com/andybalholm/cascadia"))
-    (native-inputs
-     (list go-golang-org-x-net))
-    (home-page "https://github.com/andybalholm/cascadia/")
-    (synopsis "CSS selectors for HTML")
-    (description "The Cascadia package implements CSS selectors for use with
-the parse trees produced by the html package.")
-    (license license:bsd-2)))
-
 (define-public go-github-com-puerkitobio-goquery
   (package
     (name "go-github-com-puerkitobio-goquery")
-- 
2.41.0

